kowl/templates/configmap.yaml

```diff
@@ -6,16 +6,16 @@ metadata:
     {{- include "kowl.labels" . | nindent 4 }}
 data:
   config.yaml: |
-    {{- toYaml .Values.kowl.config | nindent 4}}
+    {{- tpl (toYaml .Values.kowl.config) $ | nindent 4 }}
 
   {{- if .Values.kowl.roles }}
   roles.yaml: |
     roles:
-      {{- toYaml .Values.kowl.roles | nindent 6 }}
+      {{- tpl (toYaml .Values.kowl.config.roles) $ | nindent 6 }}
```

Comment:
Oh shoot, good fine with the config property being at the wrong level. I'm surprised it wasn't noticed before.

Reply:
@weeco 

Do this based on the existing value file.

Please verify, if this is still correct. Since `roles` is a part of the `config` hash, all settings will be written to the config.yaml, too.